app.js

```diff
@@ -1,17 +1,67 @@
 var express = require('express');
 var databaseConfig = require('./config/databaseConfig');
+var flash = require('connect-flash');
```

Comment:
flash stores messages in the session to be displayed to the user. We may decide we don't need/want this module.